packages/datetime/src/_daterangepicker.scss

```diff
@@ -14,6 +14,10 @@
     margin-right: $pt-grid-size;
   }
 
+  .DayPicker-NavButton--interactionDisabled {
+    display: none;
+  }
```

Comment:
Breaking change in v6.0.0. The element is now always rendered, but with this CSS class conditionally: https://github.com/gpbl/react-day-picker/blob/master/CHANGELOG.md#v600-2017-06-16